easybuild/easyconfigs/x/Xmipp/Xmipp-3.20.07-Boreas-fosscuda-2020b.eb

```diff
@@ -0,0 +1,81 @@
+name = 'Xmipp'
+version = '3.20.07-Boreas'
+
+homepage = 'https://scipion-em.github.io/docs/'
```

Comment:
@smoors This looks... wrong? So does the `description`

Reply:
it's not really wrong because Xmipp is part of Scipion, but I'll change it to be more precise.